src/krate.rs

```diff
@@ -953,10 +972,10 @@ pub fn owners(req: &mut Request) -> CargoResult<Response> {
     let tx = try!(req.tx());
     let krate = try!(Crate::find_by_name(tx, crate_name));
     let owners = try!(krate.owners(tx));
-    let owners = owners.into_iter().map(|u| u.encodable()).collect();
+    let owners = owners.into_iter().map(|o| o.encodable()).collect();
 
     #[derive(RustcEncodable)]
-    struct R { users: Vec<EncodableUser> }
+    struct R { users: Vec<EncodableOwner> }
```

Comment:
Note: this is the one place that I'm not sure how to handle API-wise -- this is something that cargo sends *us*. I suppose we could have two duplicate fields, both of which are Options? Try to decode both, pick whichever has stuff first?